rc/tools/git.kak

```diff
@@ -267,6 +267,13 @@ define-command -params 1.. \
 			$@
 		EOF
 
+        # Is there anything to commit?
+        staged_files = $(git diff --name-only)
+        if [ $staged_files = '']; then
```

Comment:
`$staged_files` should be in double-quotes for shell-safety, and there should be a space between `''` and `]`.

Reply:
Thanks -- cleaned it up!